Cargo.toml

```
@@ -3,7 +3,7 @@ name = "async_fn"
 authors = [
     "Daniel Henry-Mantilla <daniel.henry.mantilla@gmail.com>"
 ]
-version = "0.0.2"  # Keep in sync
+version = "0.0.3"  # Keep in sync
```

Let's aim for a version with a minor in it:
```suggestion
version = "0.1.0-rc1"  # Keep in sync
```

  - using an RC first to sanity-check rendered docs and usage by certain dependents

---

✅